pkgs/tools/package-management/appimagekit/default.nix

```diff
@@ -1,6 +1,6 @@
 { stdenv, fetchFromGitHub
 , pkgconfig, cmake, autoconf, automake, libtool
-, wget, xxd, desktop-file-utils
+, wget, xxd, desktop-file-utils, file
```

Comment:
You receive `file` argument, but you never use it. This has no effect on anything.

Reply:
Good catch, must be a leftover.